mmr-lib/Templates/T4Extensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace MMRando.Templates
+{
+    internal static class T4Extensions
+    {
+        public static MethodInfo GetMethod(this Type type, string method, params Type[] parameters)
+        {
+            return type.GetRuntimeMethod(method, parameters);
+        }
+    }
+}
+
+namespace System.CodeDom.Compiler
```

Comment:
What's this doing in here?

Reply:
It exists to resolve dependency issue. CompilerError is not implemented in System.CodeDom.Compiler in .net standard 2.0, but is generated by the t4 tool.